### PR TITLE
d2d-client: describe own device

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

**Review by ondrejtomcik — CHANGES_REQUESTED** [review]
Nice UML diagrams!

File: content/en/device-to-device-client/ownership-transfer.md [hunk 1]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
 With the default configuration, the plgd d2d client provisions the device with the pre-shared key, generated during the startup. This one is then used for every device.
 {{% /note %}}
 
+### Own device
 
+The plgd d2d client can be initialized in 3 different ways, for each of them the own device setup credential different:
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

File: content/en/device-to-device-client/ownership-transfer.md [hunk 2]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
...
 
+- [Devices and single client](../client-initialization/#devices-and-single-client)
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

File: content/en/device-to-device-client/ownership-transfer.md [hunk 3]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
...
+participant D as "Device\n(in the local network with Service)"
+
+S -> C ++: Own device
+C -> D++: Set PreSharedKey
+return
+return Device owned
+@enduml
+{{< /plantuml >}}
+
+- [Client requests the identity certificate on it's own](../client-initialization/#client-requests-the-identity-certificate-on-its-own)
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

File: content/en/device-to-device-client/ownership-transfer.md [hunk 4]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
...
+
+In the D2D client service, the identity certificate is setup for the device and signed by the certificate authority.
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

File: content/en/device-to-device-client/ownership-transfer.md [hunk 5]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
...
+return
+C -> S: Device owned
+deactivate C
+
+@enduml
+{{< /plantuml >}}
+
+- [User agent mediates CSR from the client](../client-initialization/#user-agent-mediates-csr-from-the-client)
+
+The D2D client service sets up the identity certificate to be signed by the certificate authority to the device, but the signing process is handled by the D2D client Web App.
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

File: content/en/device-to-device-client/ownership-transfer.md [hunk 6]
```diff
@@ -28,5 +28,99 @@ Supported [credetials](https://openconnectivity.org/specs/OCF_Security_Specifica
...
+return Device Identity chain
+C -> D++: Set Identity chain\n(Device Identity chain)
+return
+C -> S: Device owned
+deactivate C
+
+@enduml
+{{< /plantuml >}}
+
+- [User agent mediates CSR from the client](../client-initialization/#user-agent-mediates-csr-from-the-client)
```
[the remark this answers is not in the document]
Comment (jkralik) [reply content/en/device-to-device-client/ownership-transfer.md]: fixed

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**